__tests__/registries/npm-registry.js

```diff
@@ -197,69 +197,31 @@ describe('request', () => {
 
     npmRegistry.config = {
       '//some.other.registry/:_authToken': 'testScopedAuthToken',
-      '@testScope:registry': '//some.other.registry/',
+      '@testScope:registry': 'https://some.other.registry/',
```

Comment:
By default, `npm login` populates the `.npmrc` file with the full url, e.g. 

```
@scope:registry=https://some.other.registry/
```

and not

```
@scope:registry=//some.other.registry/
```

I made sure this is not a regression by editing away the `https://` in my `~/.npmrc` and running `npm install` which gives me `npm ERR! Only absolute URLs are supported`